package.json

```diff
@@ -18,23 +18,25 @@
   },
   "homepage": "https://github.com/cliqz-oss/android-browser#readme",
   "dependencies": {
-    "@cliqz/indexeddbshim": "3.4.1",
-    "browser-core": "http://cdn2.cliqz.com/update/edge/react-native/v3.26/3.26.2.tgz",
+    "@babel/runtime": "^7.2.0",
+    "@cliqz/indexeddbshim": "^4.1.1",
+    "browser-core": "http://cdn2.cliqz.com/update/edge/cliqz-ios/master/latest.tgz",
```

Comment:
Here's link for stabilization branch https://s3.amazonaws.com/cdncliqz/update/edge/ghostery-ios/v3.33/3.33.0.8fbc8d2.tgz

Reply:
TODO: above link will change. We need to merge a few more stuff